docs/jwst/resample/main.rst

```diff
@@ -34,10 +34,75 @@ WCS of each input image and the WCS of the defined output product.
 This mapping function gets passed to cdriz to drive the actual
 drizzling to create the output product.
 
+Context Image
+-------------
+
+In addition to image data, resample step also creates a "context image" stored
+in the ``con`` attribute in the output data model or ``'CON'`` extension
+of the FITS file. Each pixel in the context image is a bit field that encodes
+information about which input image has contributed to the corresponding
+pixel in the resampled data array. Context image uses 32 bit integers to encode
+this information and hence it can keep track of only 32 input images.
+First bit corresponds to the first input image, second bit corrsponds to the
+second input image, and so on. If the number of input images is larger than 32,
+then it is necessary to have multiple context images ("planes") to hold
+information about all input images
+with the first plane encoding which of the first 32 images contributed
+to the output data pixel, second plane representing next 32 input images
+(number 33-64), etc. For this reason, context array is a 3D array of the type
+``numpy.int32`` and shape ``(np, ny, nx)`` where ``nx`` and ``ny``
```

Comment:
not sure we need to say the type is "**numpy**.int32". Isn't just generic "int32" sufficient?

Reply:
Oh and actually isn't it *unsigned* 32-bit, i.e. uint32?

Reply:
1. `numpy.int32` is a real and actual type of the context array. There is no generic `int32` in Python.
2. I think we should modify the type of the array from `int32` to `uint32`. But since it this PR deals primarily with documenting context arrays + a convenience function, I thought to leave change of type to a separate PR. Moreover, changing type would involve modifying image model's schema.

Reply:
Sounds like a plan.